### PR TITLE
test for directChatsMessagesByChatId

### PR DESCRIPTION
**Pull request by saga2405**
i've created and tested the test for directChatsMessagesByChatID. I've followed all the guidelines for writing this test
#580

### CODE REVIEW

**Comment by saga2405** [comment]
done

**Comment by saga2405** [comment]
i've fixed the conflicting file p

File: tests/resolvers/direct_chat_query/chatsbychatID.spec.js [hunk 1]
```diff
@@ -0,0 +1,154 @@
+const axios = require('axios');
+const shortid = require('shortid');
+const { URL } = require('../../../constants');
+const getToken = require('../../functions/getToken');
+const getUserId = require('../../functions/getUserId');
+const directChatsMessagesByChatID = require('../../../lib/resolvers/direct_chat_query/directChatsMessagesByChatID');
+const database = require('../../../db');
+// jest.useFakeTimers();
```
Comment (JamaicanFriedChicken) [inline tests/resolvers/direct_chat_query/chatsbychatID.spec.js:8]: remove the commented out code

**Comment by JamaicanFriedChicken** [comment]
Can you get the test coverage to 100% from 87.5%?

**Comment by saga2405** [comment]
@JamaicanFriedChicken  yes, I've done it, now the code coverage is 100%, pushing it

**Review by JamaicanFriedChicken — APPROVED** [review]

**Pull request merged**